### PR TITLE
18-feature-enhancement-shorten-builder-module

### PR DESCRIPTION
**Pull request by uedar**
- Shorten the build function
- Import functions to construct surfaces from strings(`crystalstructure` and `facet`)

### CODE REVIEW

File: statebatch/statebatch/build.py [hunk 1]
```diff
@@ -3,16 +3,11 @@
...
-    bcc110,
-    bcc111,
     add_adsorbate,
 )
 from state_interface.state import STATE
 from ase.calculators.espresso import Espresso
+import importlib
 
 
 def build(atom_to_run, input_data, system_spec, calc_name, file_prefix=None):
```
Comment (kimrojas) [inline statebatch/statebatch/build.py:13]: change code
`system_spec.get("type") == "Surface" or system_spec.get("type") == "Adsorption"`
to
`elif system_spec.get("type") in ["Surface", "Adsorption"]:`

File: statebatch/statebatch/build.py [hunk 2]
```diff
@@ -71,18 +66,11 @@ def build_atoms_structure(atom_to_run, system_spec):
         facet = str(atom_to_run["Facet"])
```
Comment (kimrojas) [inline statebatch/statebatch/build.py:66]: Can you provide a some kind of test file ?

**Review by kimrojas — CHANGES_REQUESTED** [review]

File: statebatch/statebatch/build.py (hunk 1 above)
Comment (uedar) [reply statebatch/statebatch/build.py:13]: Thank you. I modified the code as you suggested.

File: statebatch/statebatch/build.py (hunk 2 above)
Comment (uedar) [reply statebatch/statebatch/build.py:64]: Are the test files in the `examples` directory not sufficient?
Or is it better to add test cases in `/tests/test_build.py`?

Comment (uedar) [reply statebatch/statebatch/build.py:64]: Are the test files in the `examples` directory not sufficient?
Or is it better to add test cases in `/tests/test_build.py`?

**Review by kimrojas — APPROVED** [review]
Looks good. Ok for merge. @uedar

**Pull request merged**